src/com/jalasoft/search/commons/PathHandler.java

```diff
@@ -0,0 +1,71 @@
+/*
+ *  Copyright (c) 2018 Jalasoft.
+ *
+ *  This software is the confidential and proprietary information of Jalasoft.
+ *  ("Confidential Information").  You shall not disclose such Confidential Information and shall use
+ *   it only in accordance with the terms of the license agreement you entered into with Jalasoft.
+ */
+
+package com.jalasoft.search.commons;
+
+import java.nio.file.InvalidPathException;
+import java.nio.file.Paths;
+
+/**
+ * This class will be handling the path
+ *
+ * @author rafael
+ * @version 1.0
+ */
+public class PathHandler {
+
+    private String path;
+
+    /**
+     * Constructor: if the path is empty. The path is set with current path
```

Comment:
If the path is empty then the path is set with current path

Reply:
Updated to: "If the path is empty then the path is set with the current path"
Thanks